src/marketing/engagement-cloud-data-mapping.md

```diff
@@ -0,0 +1,147 @@
+
+## Map data fields automatically
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the left panel, expand the **DOTDIGITAL** section and choose **Developer**.
+
+1. In the **Sync Settings** section under _Admin Task_, click **Automap Data Fields** <span class="btn">Run Now</span>.
+
+   ![]({% link images/images/config-engagement-cloud-developer-sync-settings.png %}){: .zoom}
+   [_Sync Settings - Admin Task_]({% link configuration/engagement-cloud/developer.md %})
+
+## Map data fields manually
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the left panel, expand **DOTDIGITAL** section and choose **Data Mapping**.
+
+1. For each field in the following sections, choose the corresponding data field from your dotdigital Engagement Cloud account.
+
+   -  Customer Information
+   -  Address
+   -  Sales
+   -  System Values<!--{%- if "Default.B2B Only" contains site.edition -%}-->
+   -  B2B Data <!--{%- endif -%}--><!--{%- if "Default.EE-B2B" contains site.edition -%}-->
+   -  Enterprise Data<!--{%- endif -%}-->
+
+   ![]({% link images/images/config-engagement-cloud-data-mapping-data-fields-mapping-customer-information.png %}){: .zoom}
+   [_Data Mapping_]({% link configuration/engagement-cloud/data-mapping.md %})
+
+## Sync Data Mappings
+
+1. In the Magento configuration, go to **DOTDIGITAL** > **Sync Settings**.
+
+1. In the _Sync_ section, set each database componen that you want to enable to `Yes`.
+
+   -  Customer Enabled
+   -  Guest Enabled
+   -  Subscriber Enabled
+   -  Order Enabled
+   -  Wishlist Enabled
+   -  Review Enabled
+   -  Catalog Enabled<!--{%- if "Default.B2B Only" contains site.edition -%}-->
+   -  [Shared Catalog]({% link catalog/catalog-shared.md %}) Enabled
+   -  [B2B Quote]({% link sales/quotes.md %}) Enabled<!--{%- endif -%}-->
+
+1. Click <span class="btn">Save Config</span>.
+
+1. Go to **System** > _Tools_ < **Cache Management**, and clear any invalid cache.
+
+1. Wait an hour to ensure that all contact data is imported into your dotdigital account.
+
+<!--{% if "Default.CE Only" contains site.edition %}-->
+![]({% link images/images/config-engagement-cloud-sync-settings-sync.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.EE Only" contains site.edition %}-->
+![]({% link images/images/config-engagement-cloud-sync-settings-sync.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.B2B Only" contains site.edition %}-->
+![]({% link images/images-b2b/config-engagement-cloud-sync-settings-sync.png %}){: .zoom}
+<!--{% endif %}-->
+[_Sync_]({% link configuration/engagement-cloud/sync-settings.md %})
+
+<!--{% if "Default.B2B Only" contains site.edition %}-->
+
+### Sync B2B Company Details
+
```

Comment:
there are a few "as described previously" mentions.  maybe give a little more info as to where it was discussed, maybe with an internal heading link?

Reply:
This is a great suggestion, and I'll catch it on the next round. Thanks